similarity/README.md

```diff
@@ -1,24 +1,82 @@
-### File Types.py 
-The file Types.py includes a Type class and functions
-that can be used to determine the type of each column in a dataset.
+# Structure
+- folder [comparing_all_tables](comparing_all_tables) 
+- file [Comparator](Comparator.py)
+- file [ComparatorByColumns](ComparatorByColumn.py)
+- file [Types](Types.py)
+- file [DataframeMetadata](DataFrameMetadata.py)
+- file [DataframeMetadataCreator](DataFrameMetadataCreator.py)
+## folder comparing_all_tables
+This folder contains two files categorical.ipynb and comparing.py.
+File comparing.py contains the first version of Comparator.
+Comparator compares all the tables together,
+so it is constructed by nested loops.
+It is very complicated, it has not a good design, and
+we do not recommend to use it. 
+
+File categorical.ipynb shows usage of comparing.py
+
+## file Comparator.py
+File contains Comparator class ComparatorType classes and DistanceFunction
+Comparator is part of the pipeline that is shown below
+![img_2.png](../images/pipeline1.png)
+You can see the visualization of the Comparator implementation
+in the picture below.
+The user sets several specific comparators for comparator
+(for example, bool, string and category)
+Each comparator will be executed.
+Specific comparator creates a number of distance tables for all bool columns. 
+Then these tables are merged by counting average for each cell.
+That will create one distance table for bool this table will be passed to
+Distance function together with a weight and table ratio, and it will count number.
+All numbers will be passed to Euclidean distance and the overall distance will be computed.
+![img.png](images/img.png)
+
+This comparator is used in main.py, test.ipynb and test_comparator.py
 
 ### File test.ipynb
 The file test.ipynb contains usage example of MetadataCreator class and 
 Comparator class. It also shows heatmaps and accuracy scores.
 
-### File DataFrameMetadata.py
-Contains Metadata Class and CategoricalMetadata class
 
-### File DataFrameMetadataCreator.py
-Contains MetadataCreator to create Metadata
+## file ComparatorByColumns
+This file contains the second implementation which was described 
+in [README.md](../README.md) The pipeline is in the picture below.
```

Comment:
```suggestion
in [README.md](../README.md). The pipeline is in the picture below.
```

Also please add an Anchor to the heading. It would look like `[README.md](../README.md#some-heading-in-readme)`

Reply:
Fixed